src/config.rs

```diff
@@ -212,7 +212,7 @@ pub struct UpdatesConfig {
     pub auto_update_interval: Duration,
 }
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Default, Debug, PartialEq)]
```

Comment:
@niklasmohrin I read another PR where you mentioned not to derive Default on Config structs, to prefer manual implementations in order to choose meaningful defaults. This was already done for the `RawDirectoriesConfig` so I figured it was fine. Thoughts?

Reply:
In general, I think that the same reasons apply here. Here, the derived value (`None`) makes sense, but I think that it would be better to have a manual implementation here too, in case other fields with non-trivial defaults get added. It's more subtle here though :thinking: 

@dbrgn I'm just gonna pass this one to you

Reply:
@niklasmohrin @dbrgn I changed it back. I was using `DirectoriesConfig` it in a weird way. I reasoned that I can safely unwrap the custom_pages_dir. See https://github.com/dbrgn/tealdeer/pull/184#discussion_r618709078